scripts/copy-prebuilts.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -eufo pipefail
+
+root="$BUILD_WORKSPACE_DIRECTORY/private/tools/prebuilt"
```

Comment:
consider parameterizing the directory in the sh_binary args too?

Reply:
Done